pom.xml

```diff
@@ -385,6 +390,10 @@
         <axiom.wso2.version>1.2.11-wso2v16</axiom.wso2.version>
         <axiom.osgi.version.range>[1.2.11, 2.0.0)</axiom.osgi.version.range>
 
+        <!-- Identity Event Handler Versions -->
+        <identity.event.handler.account.lock.version>1.8.13</identity.event.handler.account.lock.version>
+        <identity.event.handler.account.lock.version.range>[1.4.11, 2.0.0)</identity.event.handler.account.lock.version.range>
```

Comment:
Can't we increase the lower bound to 1.8.13?

Reply:
Resolved via https://github.com/wso2-extensions/identity-carbon-auth-rest/pull/260/commits/768a72f9be4837aa9d56721a87b5031a53458a4b